packages/Table/src/Table.js

```diff
@@ -26,17 +26,17 @@ export default function Table(props) {
       <sc.Thead>
         <tr>
           {ColumnDefinitions.map((columnDefinition, columnIndex) => {
-            const { header } = columnDefinition.props;
+            const { cell, header, ...moreColumnProps } = columnDefinition.props;
 
             if (typeof header === "function")
               return (
-                <sc.TH borderType={borderType} key={columnIndex}>
+                <sc.TH borderType={borderType} key={columnIndex} {...moreColumnProps}>
```

Comment:
The fact that someone already assigning a `className` to the `td`s will suddenly see it being applied to `th`s and potentially messing up the presentation makes me think that this change is a breaking one.

Will also note that going this way prevents us from supplying different sets of props for `th`s vs `td`s which may or may not turn out to be a problem. (with `className`, can still work around by using `th.className { ... }` or `td.className { ... }`

Reply:
I'm okay with introducing a breaking change. I think the use case here is most commonly going to be for selectors, so specify `th` or `td` will be acceptable for the convenience this will provide.  What would be the alternative? I can't think of anything that wouldn't be ugly, or a pretty significant change to the API.  

For more advanced use cases where different props are needed, it might be necessary to rely on passing render functions to `cell` and/or `header` which should provide a lot more flexibility (though it won't give you any control over the actual `th` or `td` elements).

Reply:
@alexzherdev After discussing some alternative approaches with @nahumzs, we landed back on this same approach that will be a breaking change (but probably a low impact one).